src/js/resp_types.js

```diff
@@ -214,6 +217,11 @@ function onPlayerStateChange(event) {
     }
 }
 
+function show_nasa(msg){
+    push_image(msg.image, 300,500);
```

Comment:
I think it would be better if you left it up to the client-side software to decide the image size, rather than pushing a size.
Edit: For this case, just hard-code these in the Web UI. I suspect I may make the sizing more dynamic in the Android client if I get around to it.

Reply:
So I should just leave it as is?

Reply:
I think the push_image message data shouldn't include the dimensions and that you should hard-code them in the UI, so that the client can decide how to render it :)

Reply:
I removed the height and width and got this.
![Screenshot](https://user-images.githubusercontent.com/54515877/78811205-7b11ad00-7997-11ea-8be7-56dc0783fabc.png)
The image is larger than the 650px max-width. Also, if other skills want to use the push image, they would probably want a custom size (not 300 x 500). What do you think I should do?